pkg/minikube/image/image.go

```diff
@@ -104,8 +105,24 @@ func WriteImageToDaemon(img string) error {
 	if err != nil {
 		return errors.Wrap(err, "getting tag")
 	}
+	glog.V(3).Infof("Writing image %v", tag)
 	_, err = daemon.Write(tag, i)
-	return err
+	if err != nil {
+		return errors.Wrap(err, "writing image")
+	}
+
+	//TODO: Make pkg/v1/daemon accept Ref too
```

Comment:
please make an Issue for any TODO and link to the TODO comment

Reply:
Haven't even filed a bug report about it yet

Reply:
Reported as https://github.com/google/go-containerregistry/pull/702